libs/frontend/ui-build/src/components/component-menu/ComponentEventEdit.vue

```diff
@@ -38,7 +38,18 @@
           :forceEdit="true"
           :immediateUpdate="true"
           @update="setEventParam(param[0], $event)"
-        />
+        >
+          <template v-if="param[1].type === ComponentArgPrimitive.Boolean" #input>
+            <Checkbox
```

Comment:
Show checkbox for boolean event params.
We don't have any boolean event params for now, but here's a preview of what it may look like:

<img width="318" alt="Screenshot 2024-01-17 at 3 44 00 PM" src="https://github.com/samatechtw/pubstudio-builder/assets/59676941/519ee533-aa86-46cd-9107-0518474df6d0">